content/en/v3/admin/guides/multi-cluster/multi-cluster-example.md

```diff
@@ -0,0 +1,260 @@
+
+---
+title: Multi-Cluster Example
+linktitle: Multi-Cluster Example
+type: docs
+description: An example how to build a multi-cluster environment using GKE/GSM/DNS
+weight: 101
+---
+ 
+This is an example on how to build a multi-cluster environment having two separate cluster repos (i.e. 'dev' and 'prod'). The steps will include building a GKE/GSM/DSN environment from scratch for both environments, and deploying projects to staging and the remote production. It will utilize DNS (`jx3rocks.com`), TLS, Let's Encrypt certificates. This example is inteded for an audience already familiar with Jenkins X operability and focuses on an example of actual commands used to build a multi-cluster  environment. Additional information regarding using Google as the provider for this example can be found under [Google Cloud Platform Prequisites](/v3/admin/guides/tls_dns/#prerequisites).
```

Comment:
```suggestion
This is an example on how to build a multi-cluster environment having two separate cluster repos (i.e. 'dev' and 'prod'). The steps will include building a GKE/GSM/DSN environment from scratch for both environments, and deploying projects to staging and the remote production. It will use DNS (`jx3rocks.com`), TLS, Let's Encrypt certificates. This example is intended for an audience already familiar with Jenkins X operability and focuses on an example of actual commands used to build a multi-cluster environment. Additional information regarding using Google as the provider for this example can be found under [Google Cloud Platform Prequisites](/v3/admin/guides/tls_dns/#prerequisites).
```